crates/cli/src/exports.rs

```diff
@@ -0,0 +1,28 @@
+use anyhow::{anyhow, Result};
+use convert_case::{Case, Casing};
+use std::path::Path;
+
+use crate::{js::JS, wit};
+
+pub struct Export {
+    pub wit: String,
+    pub js: String,
+}
+
+pub fn determine_js_exports(js: &JS, wit: &Path, wit_world: &str) -> Result<Vec<Export>> {
```

Comment:
Can we add some documentation here or rename this function to something that indicates that we are _renaming_ and _searching_ the relevant exports (e.g `process_exports`)? At a first glance it was not evident, at least to me, that here's where all the casing is happening.

Reply:
My thinking with `determine_js_exports` was it was getting a list of WIT exports and _determining_ which JS export corresponded to each of them. There isn't any renaming going on, the data structure that gets sent back is essentially `(wit_export, js_export)`. That said, I'll use a different name given the current one seems to not be clear. `process_exports` seems fine to me so I'll update to use that.